client/package.json

```diff
@@ -9,57 +9,55 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "cy:open": "cypress open",
-    "docker-build": "npm run build -- --configuration production"
+    "docker-build": "npm run build"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~13.2.2",
-    "@angular/cdk": "~13.2.2",
-    "@angular/common": "~13.2.2",
-    "@angular/compiler": "~13.2.2",
-    "@angular/core": "~13.2.2",
+    "@angular/animations": "~13.2.3",
+    "@angular/cdk": "~13.2.3",
+    "@angular/common": "~13.2.3",
+    "@angular/compiler": "~13.2.3",
+    "@angular/core": "~13.2.3",
     "@angular/flex-layout": "~13.0.0-beta.38",
-    "@angular/forms": "~13.2.2",
-    "@angular/material": "~13.2.2",
-    "@angular/platform-browser": "~13.2.2",
-    "@angular/platform-browser-dynamic": "~13.2.2",
-    "@angular/router": "~13.2.2",
-    "karma-coverage": "^2.2.0",
-    "rxjs": "~6.6.3",
+    "@angular/forms": "~13.2.3",
+    "@angular/material": "~13.2.3",
+    "@angular/platform-browser": "~13.2.3",
+    "@angular/platform-browser-dynamic": "~13.2.3",
+    "@angular/router": "~13.2.3",
+    "rxjs": "~7.5.4",
     "tslib": "^2.0.0",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~13.2.3",
-    "@angular-eslint/builder": "^13.0.1",
-    "@angular-eslint/eslint-plugin": "^13.0.1",
-    "@angular-eslint/eslint-plugin-template": "^13.0.1",
-    "@angular-eslint/schematics": "13.0.1",
-    "@angular-eslint/template-parser": "^13.0.1",
-    "@angular/cli": "~13.2.3",
-    "@angular/compiler-cli": "~13.2.2",
-    "@angular/language-service": "~13.2.2",
-    "@nrwl/cypress": "^11.3.0",
-    "@nrwl/workspace": "^11.3.0",
-    "@types/jasmine": "~3.6.0",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/node": "^13.7.4",
-    "@typescript-eslint/eslint-plugin": "5.3.0",
-    "@typescript-eslint/parser": "5.3.0",
-    "cypress": "^6.5.0",
+    "@angular-devkit/build-angular": "~13.2.4",
+    "@angular-eslint/builder": "^13.1.0",
+    "@angular-eslint/eslint-plugin": "^13.1.0",
+    "@angular-eslint/eslint-plugin-template": "^13.1.0",
+    "@angular-eslint/schematics": "13.1.0",
+    "@angular-eslint/template-parser": "^13.1.0",
+    "@angular/cli": "~13.2.4",
+    "@angular/compiler-cli": "~13.2.3",
+    "@angular/language-service": "~13.2.3",
+    "@cypress/schematic": "^1.6.0",
+    "@types/jasmine": "~3.10.3",
+    "@types/jasminewd2": "~2.0.10",
```

Comment:
This was used by Protractor and isn't needed anymore
```suggestion
```

Reply:
I was wondering about that, actually, since none of the other repos had it. I'd made a note to look into it, but you've saved me the trouble of figuring it out. 😀